climada/util/yearsets.py

```diff
@@ -22,223 +22,227 @@
 
 LOGGER = logging.getLogger(__name__)
 
-def impact_yearset(event_impacts, sampled_years=None, sampling_dict=None, correction_fac=True):
+def impact_yearset(imp, sampled_years=None, sampling_vect=None, lam= None,
```

Comment:
What does lam mean (why did you choose this name)?
```suggestion
def impact_yearset(imp, sampled_years=None, sampling_vect=None, lam=None,
```

Reply:
I used the same name the parameter (lambda) has in the np.random.poisson() function (https://numpy.org/doc/stable/reference/random/generated/numpy.random.poisson.html), but agree that this might not be the best name.

Reply:
aaa ok, lambda is fine. Maybe call it lambda then :D ?

Reply:
that seems to be a predefined function in python..

Reply:
correct! Well, then maybe leave it as is.